.travis.yml

```diff
@@ -42,7 +42,7 @@ sudo: required
 
 env:
   global:
-  - SCCACHE_VERS=0.2.12
+  - SCCACHE_VERS=0.2.13
   - DEADLINKS_VERS=0.3.0
```

Comment:
The `deadlinks` CI job seems to be broken because this version is outdated (the actual one is 0.4.1).